docs/README.md

```diff
@@ -441,6 +441,90 @@ You can expand any of the messages that matches the filter to see the full stack
 
 ![image](./images/user-guide/35-logdet.png)
 
+### SLIs, SLOs and Burn rate Alerts
+
+Cloud operations sandbox comes with several predefined SLOs(Service level objectives), that allow us to measure our users happiness. To learn more about SLIs and SLOs [visit here.](https://cloud.google.com/blog/products/devops-sre/sre-fundamentals-slis-slas-and-slos)
+
+Cloud operations suite provides **service oriented monitoring**, that means that we are configuring SLIs, SLOs and Burning Rates Alerts for a 'service'.  
+
+The first step in order to create SLO is to **ingest the data**, for GKE services  it comes out of the box, but you can also ingest additional data and [create custom metrics.](#Monitoring-and-logs-based-metrics)
+
+Then we need to **define our service**, Cloud Operations Sandbox' services are already detected since Istio automatically detects and creates services for us. But to demonstrate that you can create your own services, it also deploys custom services using [Terraform](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/terraform/monitoring).
+
+You can find all the services under [monitoring → services → Services Overview](https://cloud.google.com/stackdriver/docs/solutions/slo-monitoring/ui/svc-overview) , and you can create your own [custom service.](https://cloud.google.com/stackdriver/docs/solutions/slo-monitoring/ui/define-svc)
+
+![image](./images/user-guide/37-services-overview.png)  
+
+#### Services SLOs  
+
+The *predefined SLOs* are also deployed as part of [Terraform code](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/terraform/monitoring/04_slos.tf) and currently are for the mentioned custom services, the Istio service and Rating service.  
+
+**Custom services SLOs**
+``` 
+Custom service availability SLO: 90% of all non-4XX requests within the past 30 day windowed period return with 200 OK status 
```

Comment:
nit: Maybe reword?

> 90% of all valid (non-4xx) requests within the past 30 days return `200 OK`

Reply:
This comment applies to the others below as well

Reply:
I wanted it to be aligned with the TF.
I rephrased to align with our suggested SLO format, both in TF notes and in this page accordingly.